supply-chain/config.toml

```diff
@@ -43,7 +39,7 @@ version = "0.9.4"
 criteria = "safe-to-deploy"
 
 [[exemptions.ahash]]
-version = "0.7.6"
+version = "0.8.2"
 criteria = "safe-to-deploy"
```

Comment:
We're trying to avoid any new exceptions here, even for minor updates like this. What you can do, though, is audit the 0.7.6 to 0.8.2 diff which should be smaller than auditing the entire crate. We arbitrarily chose 0.7.6 as "safe to deploy" during the initial import, so it's ok to go relative to that.